app/src/main/java/com/quran/labs/androidquran/ui/fragment/QuranAdvancedSettingsFragment.java

```diff
@@ -98,7 +97,7 @@ public void onSuccess(String logs) {
                   Intent intent = new Intent(Intent.ACTION_SEND);
                   intent.setType("message/rfc822");
                   intent.putExtra(Intent.EXTRA_EMAIL,
-                      new String[]{ appContext.getString(R.string.logs_email) });
+                      new String[]{appContext.getString(R.string.logs_email)});
```

Comment:
nitpick - please keep the space after the `{` and before the closing `}` here and throughout :)